src/scss/abstracts/_mixins.scss

```diff
@@ -175,6 +175,7 @@
 	}
 
 	// Workshop Page
+	.disable-form,
```

Comment:
This class no longer applies to a form. Rename "disable-form" to something like "disabled-element".